t/reconfigure/consensus.t.js

```diff
@@ -31,10 +31,8 @@ function prove (async, assert) {
              -initial-cluster reconfigure-etcd=http://' + ip + ':2380 \
              -initial-cluster-state new', async())
     }, function () {
-        var wait
-        var consensus = new Consensus('test', ip, '2379', function (properties, callback) {
-            wait(null, properties)
-            callback()
+        var consensus = new Consensus('test', ip, '2379', function (callback) {
+            callback(null)
```

Comment:
Shouldn't you add an assert here to assert that it got called? The `wait` had a purpose. Not to gather the results, but to pause the test until the callback is fired.